runtime/ops/os/mod.rs

```diff
@@ -485,7 +524,7 @@ fn rss() -> usize {
   }
 }
 
-fn os_uptime(state: &mut OpState) -> Result<u64, AnyError> {
+fn os_uptime(state: &mut OpState) -> Result<u64, deno_core::error::AnyError> {
```

Comment:
Why not use concrete error types in this file?

Reply:
many ops in this file just do permission checks; for ops that only do perm checks i am leaving the error as anyerror, and then will be replaced with the concrete permission error once the permission error rework lands